packages/core/src/client.ts

```diff
@@ -890,20 +890,71 @@ export class SkipRouter {
   async transactionStatus({
     chainID,
     txHash,
+    options,
   }: {
     chainID: string;
     txHash: string;
+    options?: {
```

Comment:
Could we move this to the top level router object and make retryParams a setting in the router options? 

That way we could apply it consistently to other methods going forward